src/vanilla.ts

```diff
@@ -1,8 +1,9 @@
 type SetStateInternal<T> = {
   _(
     partial: T | Partial<T> | { _(state: T): T | Partial<T> }['_'],
-    replace?: boolean | undefined,
+    replace?: false,
```

Comment:
I see. I too thought `| undefined` is required somehow, but this is functions argument, not object property.